### ISSUE TITLE
[Issue]: loss of music playlist content after scan

### ISSUE THREAD

**Issue by 4EverChaos** [jellyfin/jellyfin#11625]
### Please describe your bug

Since 10.9/.1 are music playlists based on a m3u file without content after a fresh media library scan. And a targeted media data refresh on that specific playlist. 
The m3u playlists are automatically generated by my music library program MusicBee with an Unix address scheme. That has worked flawlessly prior the latest release. 
The weird part is that the playlists are then again filled with the missing tracks as expected, after a reboot of the Jellyfin server. (Dashboard setting)
Not sure what is causing that behaviour. 

### Reproduction Steps

1. Jellyfin 10.9/.1
2. Create m3u playlist with Unix pathing scheme
3. Scan library or refresh the metadata of the specific playlist
4. Check the playlist content - should be empty now
5. Go to the settings and restart Jellyfin
6. Check the playlist again - playlist should be filled as expected 

### Jellyfin Version

10.9.1

### if other:

10.9/.1

### Environment

```markdown
- OS: Ubuntu LTS 22.04.4
- Linux Kernel: 5.15 
- Virtualization: Proxmox VM
- Clients: Firefox, Sonixd, Finamp, Feishin web application
- Browser: Firefox, Arc Browser on Mac
- FFmpeg Version:
- Playback Method:
- Hardware Acceleration:
- GPU Model:
- Plugins:
- Reverse Proxy:
- Base URL:
- Networking:
- Storage:
```

### Jellyfin logs

```shell
If requested 
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Shadowghost** [comment]
I also ran into this and was unable to find out, why the scanner is not correctly reading the file on first scan. Just scan twice and it works.

**Comment by 4EverChaos** [comment]
Interesting. So you scan and the playlists are empty but when you scan again they are filled? You scan the whole music library or just the playlist? Because both doesn’t work for me. But when I restart Jellyfin itself it fixes the problem immediately.

**Comment by 4EverChaos** [comment]
Hm that didn’t help in my case either. Also tried it with different clients / apps. 
For now I just turned off the automatic library scan and scan when I know that changes were made, plus the reboot. But obviously that’s not the solution…

**Comment by PLK2** [comment]
I read issue was fixed in 10.9.7, but I just installed that and my m3us are still showing as empty :(

**Comment by Shadowghost** [comment]
You need to rescan once.

**Comment by felix920506** [comment]
fixed by #11673
